packages/platform-android/src/commands/runAndroid/index.ts

```diff
@@ -179,6 +219,15 @@ function runOnSpecificDevice(
         ...devices,
       );
     }
+  } else if (devices.length === 1) {
+    buildApk(gradlew);
+    installAndLaunchOnDevice(
```

Comment:
This is super similar to https://github.com/react-native-community/cli/pull/807/files#diff-2302ca84a9ef71bfb8384b1d669d2aaaR208-R216. Can we simplify it to call `installAndLaunchOnDevice` only once?

Reply:
It's also a bit confusing that we ignore `deviceId` passed to the function if the `devices.lenght` is `1`. Something to keep in mind.